doc/CONTRIBUTION-GUIDELINES.md

```diff
@@ -0,0 +1,48 @@
+# PRIVACY BADGER CONTRIBUTION GUIDELINES
+
+### GIT ETTIQUITE
+
+When contributing to via git
+
+If You are an owner:
+
+ - Checkout the main repository and make a local branch for your change
+
+ - Make your edits locally and then commit them (preferably squashed into one commit)
+   to the origin repo as branch
+
+ - Create a pull request from your branch to master, and have someone else code
+   review your change
+
+ - Once your change has been merged, delete your branch
+
+If you are not an owner (and thus can't make a branch):
+
+ - Follow the same procedure but with a personal fork of the repository
+
+ - Ensure that your fork is up to date before committing your changes
+
+### GENERAL ETTIQUITE
+
+When making a code change:
+
+ - Make sure all your tests pass, and take time to write new tests for your
+   additions
+
+ - Take time to ensure your code works as expected, effort skipped here merely
+   pushes the burden onto whomever reviews your commit
+
+ - Make sure commented lines and print statements are removed before submition
```

Comment:
I would expand this to say run jshint on your code and fix any linting errors before you commit.